lib/recipes-data/src/lib/telemetry.ts

```diff
@@ -86,9 +52,8 @@ export async function sendTelemetryEvent(
 		console.error(`sendEvent caught ${typeofErr}: ${JSON.stringify(err)}`);
 		const realAttempt = attempt ?? 1;
 		if (realAttempt < maxAttempts) {
-			console.log('Refreshing credentials and trying again');
+			console.log('Trying again');
```

Comment:
This log line, seen in isolation, requires you to go to the code to understand it; perhaps we could keep or add to the context we previously had? Or maybe the previous line is incorrect?

Reply:
By itself, I agree; however, it would always be preceded by `sendEvent caught ${typeofErr}: ${JSON.stringify(err)}` which is logged at ERROR level so I believe that it does make sense in context:

```
ERROR sendEvent caught someErrorClass: someLongDescription
INFO Trying again
```

Reply:
No guarantee those logs hit ELK in that order, but perhaps this service isn't super chatty, could be fine.